textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriterTest.java

```diff
@@ -1,6 +1,10 @@
 package edu.uci.ics.textdb.dataflow.queryrewriter;
 
+import edu.uci.ics.textdb.api.common.IField;
```

Comment:
@kishore-narendran  yes it can, could you check the file TestUtils.java 
Method is overloaded. And as @chenlica  suggested you can refer to DictionaryMatcherTest as an example.